libraries/aws_s3_buckets.rb

```diff
@@ -15,7 +15,6 @@ class AwsS3Buckets < AwsResourceBase
 
   FilterTable.create
              .register_column(:bucket_names, field: :bucket_name)
-             .register_column(:tags,         field: :tags)
```

Comment:
```suggestion
            .register_column(:tags, field: :tags, lazy: ->() { |row,_c,_t| row[:tags] = fetch_tags(row[:bucket_name]) })
```

Reply:
@sathish-progress - I've tried this already. But the syntax gives this error 
```
/var/lib/gems/2.7.0/gems/inspec-core-4.52.9/lib/inspec/profile_context.rb:170:in `instance_eval': libraries/aws_s3_buckets.rb:18: syntax error, unexpected '|' (SyntaxError)
...s, field: :tags, lazy: ->() { |row,_c,_t| row[:tags] = fetch...
...                              ^
libraries/aws_s3_buckets.rb:18: syntax error, unexpected '|', expecting '='
...:tags, lazy: ->() { |row,_c,_t| row[:tags] = fetch_tags(row[...
...
```

Additionally, it seems that instance methods are not accessible in the lazy load of a filter table